code/modules/spells/spell_types/spacetime_distortion.dm

```diff
@@ -73,6 +76,7 @@
 /obj/effect/cross_action/spacetime_dist/New()
```

Comment:
Basically, just always use Initialize() instead of New() for everything. Initialize is the same, except the first arg is "mapload" which is if the atom's being made on mapload.

Apparently Byond New() is bad, so we're using our own proc, with hookers and blackjack